assets/css/admin-tests.css

```
@@ -0,0 +1,42 @@
+@import url('multi-range-slider.css');
```

Stylelint (17.14.0)

[error] Expected "url('multi-range-slider.css')" to be "'multi-range-slider.css'" (import-notation)

(import-notation)